src/bind.js

```diff
@@ -18,8 +18,10 @@
  *
  * @return {Function}
  */
-function bind(callback) {
-  // write code here
+function bind(callback, ...params) {
+  return (...paramsToBind) => {
```

Comment:
write it in one line
```suggestion
  return (...paramsToBind) => callback(...params, ...paramsToBind);
```